src/macaron/dependency_analyzer/cyclonedx.py

```diff
@@ -114,8 +104,8 @@ class DependencyInfo(TypedDict):
     available: SCMStatus
 
 
-class DependencyAnalyzer(ABC):
-    """This abstract class is used to implement dependency analyzers."""
+class DependencyAnalyzer:
+    """This base class is used to implement dependency analyzers."""
```

Comment:
From my understanding, `DependencyAnalyzer` deals with CyclondDx SBOM only. Could we add this as part of the doc string here ?